workflow/snakemake_rules/main_workflow.smk

```diff
@@ -1277,16 +1277,33 @@ def _get_node_data_by_wildcards(wildcards):
 
     return inputs
 
+rule build_description:
+    message: "Templating build description for Auspice"
+    input:
+        description = lambda w: config["builds"][w.build_name]["description"] if "description" in config["builds"][w.build_name] else config["files"]["description"]
+    output:
+        description = "results/{build_name}/description.md"
+    log:
+        "logs/build_description_{build_name}.txt"
+    conda: config["conda_environment"]
+    shell:
+        """
+        env BUILD={wildcards.build_name:q} \
+            perl -pe 's/\$\{{BUILD\}}/$ENV{{BUILD}}/g' \
+                < {input.description:q} \
+                > {output.description:q}
+        """
```

Comment:
Revisiting this now, it seems like a perfect candidate for [the new template rendering functionality in Snakemake 7](https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#template-rendering-integration). Something to keep in mind, if we find ourselves reaching for this kind of functionality again...